server/endpoints/Profile.ts

```diff
@@ -0,0 +1,36 @@
+import { body } from "express-validator";
+import { Context, Endpoint } from "../endpoints";
+import { ReviewDocument, Reviews, Students } from "../dbDefs";
+
+// The type of a query with a studentId
+export interface NetIdQuery {
+  netId: string;
+}
+
+export const getTotalLikesByStudentId: Endpoint<NetIdQuery> = {
+  guard: [body("netId").notEmpty().isAscii()],
+  callback: async (ctx: Context, request: NetIdQuery) => {
+    const { netId } = request;
+    let totalLikes = 0;
+    try {
+      const studentDoc = await Students.findOne({ netId });
+      const reviewIds = studentDoc.reviews;
+      const reviews: ReviewDocument[] = await Promise.all(
+        (reviewIds).map(async (reviewId) => await Reviews.findOne({ _id: reviewId })),
+      );
+      reviews.forEach((review) => {
+        if ('likes' in review) {
+          totalLikes += review.likes;
+        }
+      });
+
+      return { resCode: 1, totalLikes };
```

Comment:
Lets start standardizing the return object of our end points. In Michelle's PR that was merged in earlier (https://github.com/cornell-dti/course-reviews-react-2.0/pull/280/files) we went with `{code: <> , message:<>}` where message contains data if there is no error. Lets try that here too

Reply:
fixed!